### DISCUSSION TITLE
Initial config files for supervised ID tracking?

### DISCUSSION THREAD

**Discussion by irieth** [talmolab/sleap#1497] [Ideas]
I am currently using the GUI to track multiple mice with distinct coat markings. I have been successful in training both bottom-up and top-down models without the supervised ID, but am having a bit of trouble with training the supervised ID models. I found initial configuration files for these models here: (https://github.com/talmolab/sleap/tree/e4fca4f96b711648b13f4e572995e77d0a718ead/tests/data/models/min_tracks_2node.UNet.bottomup_multiclass), but these were quite tucked away within the Github. Is there a more updated version of these initial configs? Any help would be greatly appreciated!

**Comment by irieth** [comment]
Here are the configs that I have been using, which has yielded some success 
![bottom_up_multiclass_training_config_settings](https://github.com/talmolab/sleap/assets/69976818/f443abe3-e3d4-4df0-9023-e2a8622a7ba6)
![bottom_up_multiclass_training_config_metrics](https://github.com/talmolab/sleap/assets/69976818/ee408b26-01ca-4586-a965-fced0e308a01)

**Comment by talmo** [comment]
Hi @irieth,

Thanks for the reminder -- we hadn't gotten around to actually exposing the ID configs.

Those settings look great, and I'm glad they're working for you!

The only thing we've found in the past that might help is to set the second output stride of the head to a number higher than the first one (ideally the highest one available that corresponds to your max stride of 32). This is because identification works better with larger-scale features, while landmark localization benefits from having higher resolution feature maps.

In any case, I'm converting this to an Idea so we can follow up on this by creating some training profiles for the ID models.

Thanks!!

Talmo
